tox.ini

```diff
@@ -14,16 +14,23 @@ deps =
 commands =
   black {toxinidir}/plugins {toxinidir}/tests
 
+[testenv:ansible-lint]
+deps =
+  ansible-lint>=6.7.0
+commands = ansible-lint --profile production --format pep8 --nocolor --strict {toxinidir}/plugins {toxinidir}/roles
+
 [testenv:linters]
 deps =
   yamllint
   flake8
   {[testenv:black]deps}
+  {[testenv:ansible-lint]deps}
 
 commands =
   black -v --check --diff {toxinidir}/plugins {toxinidir}/tests
   yamllint {toxinidir}
   flake8 {toxinidir}
+  ansible-lint --profile production --format pep8 --nocolor --strict {toxinidir}/plugins {toxinidir}/roles
```

Comment:
You can also use the substitution here if you want, like `{[testenv:ansible-lint]commands}`

Reply:
Right, thank you! Cleaner this way!